test/integration/bytea.test.ts

```diff
@@ -0,0 +1,48 @@
+import { Firebolt } from "../../src/index";
+
+const connectionParams = {
+  auth: {
+    username: process.env.FIREBOLT_USERNAME as string,
+    password: process.env.FIREBOLT_PASSWORD as string
+  },
+  database: process.env.FIREBOLT_DATABASE as string,
+  engineName: process.env.FIREBOLT_ENGINE_NAME as string
+};
+
+jest.setTimeout(100000);
+
+describe("bytea", () => {
+  it("handles select bytea", async () => {
+    const firebolt = Firebolt({
```

Comment:
we also need a way to serialize value of bytea type.
I have something here, but it was not tested: https://github.com/firebolt-db/firebolt-node-sdk/blob/main/src/formatter/index.ts#L261
Maybe also to test if it suit or sql syntax?

Reply:
Sure I will have a look 👍

Reply:
Doesn't look like the current `escapeBuffer()` will work.

One way is to generate a string that looks like `'\\x616263'::BYTEA`.
The double backslash is needed when the default settings `standard_conforming_strings=0` are used.
But if using `standard_conforming_strings=1` then there must only be one backslash.

See https://docs.firebolt.io/general-reference/bytea-data-type.html#literal-string-interpretation

Another way would be to generate something like `DECODE('616263', 'hex')`.
This works the same no matter the setting of `standard_conforming_strings`.
But I don't know if this is slower than the above method.

Reply:
I changed it to

``` 
  private escapeBuffer(param: Buffer) {
    return this.escapeString("\\x" + param.toString("hex"));
  }
  ```
  
  And it seems alright (added this test: https://github.com/firebolt-db/firebolt-node-sdk/blob/23197e9b1575b6adec7918a925461fb1d4b28e3b/test/unit/statement.test.ts#L237-L244)

Reply:
I'm not sure, but don't you want something like `return this.escapeString("\\x" + param.toString("hex") + "::BYTEA");`? Otherwise it's just a string to the database.

For example `SELECT ENCODE('\\x61', 'hex')` is an error, but `SELECT ENCODE('\\x61'::BYTEA, 'hex')` works.

Reply:
I don't think we should append with ::bytea. The goal is just to replace the param marker (?) with the buffer value specified. In the example (in the test) we actually check that the bytea value of hello_world equals the value specified (68656c6c6f5f776f726c64). If the user wanted to have it appended with ::bytea they could do
SELECT ?::bytea